styles/starwarsffg.css

```diff
@@ -1773,6 +1773,10 @@ img {
 .starwarsffg .item-sheet-gear .charname input {
   color: #2c0344;
 }
+.starwarsffg .item-sheet-gear .profile-img {
+  height: 124px;
+  width: 100%;
+}
```

Comment:
These changes will get overridden the next time the SCSS files are compiled.

Reply:
If you don't mind, for now, merge it and I'll create an issue to pull this and the other things you've commented into the SCSS files (since there are other changes.)

Unless each release recompiles them, in which case... I have made a mistake

Reply:
Works for me... we don't do any automated builds on this repo, so manual recompile only. 👍